lib/api/core/models/repositories/profileRepository.js

```diff
@@ -2,6 +2,7 @@ module.exports = function (kuzzle) {
   var
     _ = require('lodash'),
     q = require('q'),
+    extend = require('util')._extend,
```

Comment:
I can't find any `_extend` method in the NodeJS v4 `util` (API documentation)[https://nodejs.org/dist/latest-v4.x/docs/api/util.html].
I see it in the (v6 API documentation)[https://nodejs.org/api/util.html#util_util_extend_obj], and it is already marked as **deprecated**.
Why don't you use lodash's `extend` method?